### ISSUE TITLE
alltoall and alltoalls unit tests fail with 3 or more PEs

### ISSUE THREAD

**Issue by davidozog** [Sandia-OpenSHMEM/SOS#1035]
Output for 3 PEs is:
```
A2A: pe_start = 0, pe_stride = 1, pe_size = 2
[0] out[1] = 1, expected 2
```

**Comment by wrrobin** [comment]
Can you check this one if its resolved? @kholland-intel

**Comment by kholland-intel** [comment]
@wrrobin This issue has been resolved by PR #1037. We are now getting correct results for unit tests `alltoall` and `alltoalls` with 3+ PEs.

Thanks, @davidozog for the fix!
